wgsl/index.bs

```diff
@@ -4926,37 +4926,37 @@ See [[#sync-builtin-functions]].
     <th>Preconditions<th>Conclusions<th>Semantics
   </thead>
   <tr algorithm="matrix addition">
-    <td rowspan=2>|e1|, |e2|: mat|M|x|N|&lt;f32&gt
-    <td>|e1| `+` |e2|: mat|M|x|N|&lt;f32&gt<br>
+    <td rowspan=2>|e1|, |e2|: mat|R|x|C|&lt;f32&gt
```

Comment:
This should be mat|C|x|R|

This was definitely a pre-existing problem, and would be a big source of confusion!

Reply:
Was wondering why those were reversed 😅